packages/shaper-utils/src/FileUtils.ts

```diff
@@ -119,3 +119,53 @@ export const FileUtils = {
   transformFiles,
   resolvePaths,
 };
+
+export type Plugin = (args: Array<string | number>) => Promise<any>;
+
+export function getPlugin(name: string, dir = 'node_modules'): Plugin  {
```

Comment:
Instead of direct export here, can we export (all functions) under FileUtils? That way if we need multiple functions in other modules we only need to import FileUtils once.